src/ChainOfResponsability/Demo.java

```diff
@@ -0,0 +1,35 @@
+package ChainOfResponsability;
+
+import java.util.HashMap;
+
+import ChainOfResponsability.Context.Input;
+import ChainOfResponsability.Context.Output;
+import ChainOfResponsability.Handler.DateVerifier;
+import ChainOfResponsability.Handler.DigitVerifier;
+import ChainOfResponsability.Handler.EmailVerifier;
+import ChainOfResponsability.Handler.ISanitizer;
+
+public class Demo {
+    public static void main(String[] args) {
+        HashMap<String, String> inputs = new HashMap<>();
+        inputs.put("email", "example@gmail.com");
+        inputs.put("digit", "123456");
+        inputs.put("date", "2021-01-01");
+
+        Input fields = new Input(inputs);
+
+        ISanitizer emailVerifier = new EmailVerifier(inputs.get("email"));
```

Comment:
this is odd, why do you need to pass values here ? the whole intention is to send `inputs ` to the sanitizer to provide a sanitized version of the output with the appropriate types